ocean_lib/web3_internal/contract_base.py

```diff
@@ -192,7 +193,7 @@ def send_transaction(
         contract_fn = getattr(self.contract.functions, fn_name)(*fn_args)
         contract_function = CustomContractFunction(contract_fn)
         _transact = {
-            "from": from_wallet.address,
+            "from": ContractBase.to_checksum_address(from_wallet.address.lower()),
```

Comment:
why do we need our own checksumming function if it already exists in web3?

Reply:
I updated it to include `lower()` as well